core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -87,6 +91,24 @@ angular.module('oppia').component('preferencesPage', {
         _saveDataItem('user_bio', userBio);
       };
 
+      ctrl.checkBioChanged = function() {
```

Comment:
registerBioUpdate or registerBioChanged*

Reply:
How does preferenceUpdate sound?

Reply:
Since it also waits for other HTTP req also (like for saving email etc…). See func `_saveDataItem` in line 68

Reply:
`preferenceUpdated` sounds like a boolean variable (not a function name). I would suggest either have `registerBioChanged` (this function name will allow us to use this function for handling all the things required to do when a bio is changed.) or something more specific registerPreventReloadEvent or enablePreventReloadEvent. (I would prefer the first one!)

Reply:
Oh! Sorry the above 2 comments were meant for ` ctrl.allowreload` variable. Placed it in the wrong conversation.

Reply:
DONE